act/qc/__init__.py

```diff
@@ -19,15 +19,16 @@
         'bsrn_tests',
         'comparison_tests',
         'qcfilter',
-        'qctests',
```

Comment:
Why did you remove qctests here?

Reply:
@mgrover1 we are doing direct imports of QCTests so it should be available.  Do we need to keep it in the lazy_loader section if we are already importing it?

```
from .qcfilter import QCFilter
from .qctests import QCTests
```

Reply:
Good point - it looks like it is showing up in the built documentation in the github action log